BluService/PowerShellRunspace.cs

```diff
@@ -67,7 +67,7 @@ public string ExecuteScript(string scriptFile)
                 return "Exit1:runspace has exited, dispose and recreate";
             }
             _runspaceConsole.StandardInput.WriteLine(scriptFile);
-            if (!_eventWaitHandle.WaitOne(300000))
+            if (!_eventWaitHandle.WaitOne(TimeSpan.FromHours(4)))
```

Comment:
The SQL install takes a long time and was timing out, this 4 hours is intended as a preventative measure, keeping a runspace from getting stuck indefinitely.